linux/LinuxProcess.h

```diff
@@ -182,6 +213,19 @@ void LinuxProcess_printDelay(float delay_percent, char* buffer, int n);
 
 void LinuxProcess_writeField(Process* this, RichString* str, ProcessField field);
 
+/*
+TASK_COMM_LEN is defined to be 16 for /proc/[pid]/comm in man proc(5), but it is
+not available in an userspace header - so define it.
+*/
+#define TASK_COMM_LEN 16
```

Comment:
No need to export this, please keep in `linux/LinuxProcess.c`.

Reply:
Agreed